text/0079-cdk-2.0.md

```diff
@@ -200,6 +178,34 @@ auditable by users, who will naturally retain the ability to opt out of this fea
 
 [version reporting]: https://docs.aws.amazon.com/cdk/latest/guide/cli.html#version_reporting
 
+## Releasing to Package Managers
+
+`aws-cdk-lib` will be released to the various package managers under the following package name.
+
+| Github & NPM | Maven Central | PyPI | NuGet |
+| -- | -- | -- | -- |
+| `aws-cdk-lib` | `software.amazon.awscdk.lib` | `aws-cdk-lib` | `Amazon.CDK.Lib` |
+
+CDKv2 will be published in three phases - **alpha**, **developer preview** and finally **generally available**.
+During the first two phases, packages will be published with the base version of `2.0.0` and every release will use a
+sequentially increasing pre-release identifier.
+The following table describes the versioning scheme in the different package managers across the different phases.
+
+| | Github & NPM | Maven Central | PyPI | NuGet |
+| -- | -- | -- | -- | -- |
+| **Spec** | [semantic versioning] | [pom version order] | [python packaging] | [nuget versioning] |
+| **Alpha** | `2.0.0-alpha.1` <br/> `2.0.0-alpha.2` | `2.0.0-alpha.1` <br/> `2.0.0-alpha.2` | `2.0.0.a1` <br/> `2.0.0.a2` | `2.0.0-alpha.1` <br/> `2.0.0-alpha.2` |
+| **Dev Preview** | `2.0.0-devpreview.1` <br/> `2.0.0-devpreview.2` | `2.0.0-devpreview.1` <br/> `2.0.0-devpreview.2` | `2.0.0.rc1` <br/> `2.0.0.rc2` | `2.0.0-alpha.1` <br/> `2.0.0-alpha.2` |
```

Comment:
Why is NuGet still using `alpha.#` at the dev-preview stage?

Why are we not using `-rc#`/`.rc#` for all package managers at this point (so they're uniform w/ Python)?

Reply:
> Why is NuGet still using `alpha.#` at the dev-preview stage?

That was a miss. I'll fix it up.

> Why are we not using `-rc#`/`.rc#` for all package managers at this point (so they're uniform w/ Python)?

I'm thinking the same. I kept developer preview because that's the language we use everywhere else. Is there a case to be made for keeping it developer preview everywhere (probably not possible with python)